scripts/create-packages.py

```diff
@@ -53,6 +60,96 @@ def build_pex_pex(
     return output_file
 
 
+def build_pex_scies(
+    scie_dest_dir: Path, verbosity: int = 0, env: Optional[Dict[str, str]] = None
+) -> Iterator[tuple[Path, str]]:
+    scie_config = ScieConfig.load()
+
+    pex_requirement = f".[{','.join(scie_config.pex_extras)}]"
+
+    lock = PACKAGE_DIR / "pex-scie.lock"
+    if not lock.exists():
+        raise SystemExit(
+            f"The Pex scie lock at {lock} does not exist.\n"
+            f"Run `tox -e gen-scie-platform -- --all ...` to generate it."
+        )
+
+    missing_platforms: list[str] = []
+    platforms: list[tuple[str, Path]] = []
+    for platform in scie_config.platforms:
+        complete_platform = PACKAGE_DIR / "complete-platforms" / f"{platform}.json"
+        if not complete_platform.exists():
+            missing_platforms.append(platform)
+        else:
+            platforms.append((platform, complete_platform))
+    if missing_platforms:
+        missing = "\n".join(
+            f"{index}. {missing_platform}"
+            for index, missing_platform in enumerate(missing_platforms, start=1)
+        )
+        raise SystemExit(
+            f"Of the {len(platforms)} expected Pex scie complete platforms, "
+            f"{len(missing)} {'is' if len(missing) == 1 else 'are'} missing:\n{missing}"
+        )
+
+    for platform, complete_platform in platforms:
+        dest_dir = safe_mkdtemp()
+        output_file = os.path.join(dest_dir, "pex")
+        args = [
+            sys.executable,
+            "-m",
+            "pex",
+            *["-v" for _ in range(verbosity)],
+            "--disable-cache",
+            "--no-build",
+            "--no-compile",
+            "--no-use-system-time",
+            "--venv",
+            "--no-strip-pex-env",
+            "--complete-platform",
+            str(complete_platform),
+            "--lock",
+            str(lock),
+            "--scie",
+            "eager",
+            "--scie-only",
+            "--scie-name-style",
+            "platform-file-suffix",
+            "--scie-platform",
+            platform,
+            "--scie-pbs-release",
+            scie_config.pbs_release,
+            "--scie-python-version",
+            scie_config.python_version,
+            "--scie-pbs-stripped",
+            "--scie-hash-alg",
+            "sha256",
+            "--scie-busybox",
+            "@pex",
+            "--scie-busybox-pex-entrypoint-env-passthrough",
```

Comment:
N.B.: This means the released Pex scies, unlike the Pex PEX, support all 3 top-level Pex command 1st class if you leave the file name as-is. For example:
```console
:; dist/pex-linux-x86_64
Error: Could not determine which command to run.

Please select from the following boot commands:

pex
pex-tools
pex3

You can select a boot command by setting the SCIE_BOOT environment variable or else by passing it as the 1st argument.
```

So you can:
```console
# Run `pex`:
:; dist/pex-linux-x86_64 pex ...

# Run `pex3`:
:; dist/pex-linux-x86_64 pex3 ...

# Run `pex-tools`:
:; dist/pex-linux-x86_64 pex-tools ...
```

Alternatively you can do something like:
```console
# Install the binaries with top-level names (the `--symlink` is optional):
:; SCIE=install dist/pex-linux-x86_64 --symlink bin

# Run `pex`:
:; bin/pex ...

# Run `pex3`:
:; bin/pex3 ...

# Run `pex-tools`:
:; bin/pex-tools ...
```

Finally, if you re-name the downloaded Pex scie to `pex`, it behaves ~just like the Pex PEX and you must use env vars to access the alternate entry points; e.g: `PEX_SCRIPT=pex3 pex ...`.